src/payment/__snapshots__/PaymentPage.test.jsx.snap

```diff
@@ -10780,7 +10780,7 @@ exports[`<PaymentPage /> Renders correctly in various states should render the b
                 </span>
                 <span>
                   <span>
-                    $0.00
+                    -$0.00
```

Comment:
Everything looks good except these two snapshots where $0.00 becomes -$0.00. Seems unrelated, but worth looking into.

Reply:
Oh just checked travis. It looks like you just need to update these snapshots (maybe there was a problem before that is now fixed.)

Reply:
Added `<LocalizedPrice amount={discounts ? discounts * -1 : 0} />` to prevent passing -0 to LocalizedPrice